webpack/JobWizard/JobWizard.js

```diff
@@ -102,23 +107,50 @@ export const JobWizard = () => {
           timeoutToKill: execution_timeout_interval || '',
           templateValues: advancedTemplateValues,
           description: generateDefaultDescription() || '',
-          isRandomizedOrdering: false,
+          isRandomizedOrdering: randomized_ordering,
+          sshUser: ssh_user || '',
+          timeSpan: concurrency_control.time_span || '',
+          concurrencyLevel: concurrency_control.level || '',
         };
       });
     },
     [category.length]
   );
+  useEffect(() => {
+    if (rerunData) {
+      setCategory(rerunData.job_category);
+      setJobTemplateID(rerunData.template_invocations[0].template_id);
+      setDefaults({
+        data: {
+          effective_user: {
+            value: rerunData.template_invocations[0].effective_user,
+          },
+          job_template: {
+            execution_timeout_interval: rerunData.execution_timeout_interval,
+          },
+          randomized_ordering: rerunData.targeting.randomized_ordering,
+          ssh_user: rerunData.ssh_user,
+          concurrency_control: rerunData.concurrency_control,
+        },
+      });
+      setFills({
+        search: rerunData.targeting.search_query,
+        ...rerunData.inputs,
+      });
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Can this effect be eliminated and instead calculate all the values during render?

https://beta.reactjs.org/learn/you-might-not-need-an-effect#how-to-remove-unnecessary-effects

Reply:
If you feel strongly about keeping this an effect, at least the rule shouldn't be disabled.  But as I was saying before, I think the better solution is to not use an effect at all.